Backpack/Calendar/Classes/BPKCalendarPriceLabelCell.m

```diff
@@ -80,9 +82,24 @@ - (void)prepareForReuse {
 - (void)configureWithData:(id)data {
     NSAssert([data isKindOfClass:BPKCalendarPriceLabelCellData.class], @"BPKCalendarPriceLabelCell can only be configured with data of type BPKCalendarPriceLabelCellData");
     BPKCalendarPriceLabelCellData *priceLabelCellData = (BPKCalendarPriceLabelCellData *)data;
+    self.priceLabelCellData = priceLabelCellData;
     self.priceLabel.text = priceLabelCellData.price;
     self.priceLabel.textColor = priceLabelCellData.labelStyle.textColor;
     self.priceLabel.fontStyle = priceLabelCellData.labelStyle.isBold ? BPKFontStyleTextXsEmphasized : BPKFontStyleTextXs;
 }
 
+- (void)setAccessibilityLabel:(NSString *)accessibilityLabel {
+    self.consumerAccessibilityLabel = accessibilityLabel;
+}
+
+- (NSString *)accessibilityLabel {
+    NSString *accessibilityLabel = self.consumerAccessibilityLabel;
+
+    if (self.priceLabelCellData.price != nil) {
+        accessibilityLabel = [NSString stringWithFormat:@"%@ %@", self.consumerAccessibilityLabel, self.priceLabelCellData.price];
```

Comment:
I'm not sure about this. AFAIK naive concatenation of strings doesn't work well in all languages and the ordering/structure of sentence is a localisation concern.

Reply:
I guess we need to provide a delegate method so that consumers can provide a string, but maybe this is a good default?

Reply:
Yeah that's probably a good idea. I suppose this might be a good first step yes

Reply:
Although, what if the consumers have already localised the `accessibilityLabel` with the price?

Reply:
I guess we need to configure this to only append the price if the consumer isn't setting a custom value?